Sources/App/configure.swift

```diff
@@ -50,16 +52,41 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
 }
 
 fileprivate func dbConfig(environment: Environment) throws -> PostgreSQLDatabaseConfig {
+    // Configure a database
+    let databaseConfig: PostgreSQLDatabaseConfig
+    let hostname = Environment.get("DATABASE_HOSTNAME") ?? "localhost"
+    let username = Environment.get("DATABASE_USER") ?? "vapor"
+    let password: String? = Environment.get("DATABASE_PASSWORD") ?? nil
+    let databaseName: String?
+    let databasePort: Int
+    
```

Comment:
Could you add a "catchall" for a `DB_URL` env var? I personally prefer using 1 env var for my DB Config

Reply:
Done https://github.com/vapor/bot-github/pull/11/files#diff-cc9aa3d707deffc5f5cb05865ea525c7R65

Reply:
Awesome!